### ISSUE TITLE
Not possible to close browser in full screen PWA

### ISSUE THREAD

**Issue by ormejere** [ionic-team/capacitor#756]
This issue applies to a PWA running in full screen mode (opened from the home screen).

When using the Browser plugin to open a url as described in the docs (https://capacitor.ionicframework.com/docs/apis/browser):

```javascript
import { Plugins } from '@capacitor/core';
const { Browser } = Plugins;
await Browser.open('http://capacitor.ionicframework.com/');
```

The url is opened correctly but it is not possible to close it because there are no visible browser controls, close button etc.

Ideally, there would be a close (X) button in the corner or similar to allow the user to close the window and continue using the PWA.

**Comment by jcesarmobile** [comment]
You are running it as PWA, but where? iOS device? Android device? desktop browser? In chase it's a desktop browser, which one? version? Operating system?

**Comment by ormejere** [comment]
I'm using an iPhone 7+ running iOS 11.4.  It works fine in the Safari browser on the iPhone because I have access to tabs there so can close the tab but when i select "Add to Home Screen" and then run it from the home screen icon there is no way (that I can see) to close the tab.

Likewise running in the desktop browser is not a problem because the tabs are displayed and can be closed/switched between.

This is also not a problem on Android because when running from the home screen on Android, a title bar is displayed with an X icon to close the window.

**Comment by jcesarmobile** [comment]
This is really an iOS bug and I don't think it's something we can fix.

Looks like they have fixed it in 12.2 (still beta) by adding some sort of In-app browser
